R/estimateParams.R

```diff
@@ -53,3 +53,68 @@ prepareData <- function(data) {
 
   as.data.frame(dataNew[, -which(names(dataNew) == "time")], row.names = seq_len(nrow(dataNew)))
 }
+
+# transition <- weibull_transition(h01 = 0.5, h02 = 1.3, h12 = 1.1, p01 = 1.2, p02 = 0.5, p12 = 1.4)
+# trial <- getOneClinicalTrial(nPat = c(5000),
+#                             transitionByArm = list(transition),
+#                             dropout = list(rate = 0.3, time = 1),
+#                             accrual = list(param = "intensity", value = 100))
+
+# estimateParams(trial, "Weibull")
+
+NegLogLik <- function(data, params, family = c("exponential", "Weibull")) {
```

Comment:
so you could omit the `family` argument here, but instead accept `trans` - it is used in the function body but not defined at the moment

Reply:
I have now omitted `family`. I left `trans` as it is since it is a column of `data`.